indico/modules/events/editing/client/js/components/Timeline/ReviewForm.jsx

```diff
@@ -58,20 +63,28 @@ export default function ReviewForm({block}) {
 
   const InputComponent = commentFormVisible ? FinalTextArea : FinalInput;
   const inputProps = commentFormVisible ? {autoFocus: true} : {};
+  const addComment = async formData => {
+    if (commentFormVisible) {
```

Comment:
what's the reason for this check? can you even submit the form when it's not visible?

Reply:
Nope, it shouldn't be possible. I just put it here for precaution (even thouigh it should never happen)